run_tests.sh

```diff
@@ -50,7 +56,15 @@ for directory in "${SCRIPT_DIR}"/third_party/*; do
 done
 
 if $use_clang_completer; then
-  nosetests -v "${SCRIPT_DIR}/ycmd"
+  if [ "$*" ]; then
+    nosetests -v $@
```

Comment:
This is peculiar to me; I'd assume we want to pass the path to ycmd to `nosetests` always, and that the user only adds new args. In which case would the user not want the path to ycmd passed?